src/clib/pio_darray.c

```diff
@@ -821,8 +821,9 @@ PIOc_write_darray(int ncid, int varid, int ioid, PIO_Offset arraylen, void *arra
     bufptr = (void *)((char *)wmb->data + arraylen * iodesc->mpitype_size * wmb->num_arrays);
     if (arraylen > 0)
     {
+        PLOG((3, "copying %ld bytes of user data %d", arraylen * iodesc->mpitype_size, iodesc->mpitype_size));
+        printf(( "copying %ld bytes of user data %d num_arrays %d bufptr %x array %x\n", arraylen * iodesc->mpitype_size, iodesc->mpitype_size, wmb->num_arrays, bufptr, array));
```

Comment:
I'm sure you don't want to leave printf statements in...

Is the logging not working for you? Build with --enable-logging and you can add log statements without using printf...

Reply:
I'm confused, this line was already removed.  https://github.com/jedwards4b/ParallelIO/blob/refactor_subset_method/src/clib/pio_darray.c#L825

Reply:
logging works but sometimes while debugging I find it easier to enable a single print instead of all of the level 3 prints